gto/api.py

```diff
@@ -19,6 +19,14 @@
 from gto.tag import parse_name_reference
 
 
+def is_gto_repo(repo: Union[str, Repo]):
+    """Check if repo is a gto repo"""
+    try:
+        return GitRegistry.from_repo(repo).is_gto_repo()
+    except NoRepo:
+        return False
```

Comment:
Looks a bit strange to me having this in a public API, TBH.

Reply:
Making that method private since I think it's some extra case not needed for every GTO API user.